build.gradle

```diff
@@ -808,10 +809,9 @@ if (userEnableTestCoverage) {
     executionData.from = javaProjects.jacocoTestReport.executionData
 
     reports {
-      html.enabled = true
-      xml.enabled = true
+      xml.required = true
```

Comment:
Is there a reason to have two duplicated lines?

Reply:
No reason l will fit it now

Reply:
@ex172000  I did not only duplicate the xml configuration I also removed the html configuration and this is fixed now

Reply:
Nice, thank you for the explanation! New to the project so asked to learn :)